packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoiceDiscount/index.mjml

```diff
@@ -73,8 +74,8 @@ can obtain one at http://mozilla.org/MPL/2.0/. %> <%- include
       </span>
     </mj-text>
 
-    <%- include ('/partials/viewInvoice/index.mjml') %> <%- include
-    ('/partials/paymentProvider/index.mjml') %>
+    <%- include ('/partials/viewInvoice/index.mjml') %>
+    <%- include ('/partials/paymentProvider/index.mjml') %>
```

Comment:
I know the formatting is awful, but it's because this fix will add a line between the View Invoice link and Payment Method.

<img width="1129" alt="Screen Shot 2022-02-07 at 4 19 42 PM" src="https://user-images.githubusercontent.com/28129806/152873856-fb81bd97-c324-4728-b203-0425a85fcddb.png">